libraries/fc/include/fc/log/appender.hpp

```diff
@@ -3,7 +3,11 @@
 #include <fc/shared_ptr.hpp>
 #include <fc/string.hpp>
 
+#if BOOST_VERSION >= 106600
```

Comment:
This sets a pretty fragile precedent however, I think a better long term solution is lower priority at this point than compatibility with boost 1.66.  I will put up a lower priority issue for a refactor and approve this.